client/verta/verta/endpoint/_endpoint.py

```diff
@@ -126,8 +125,8 @@ def get_current_build(self):
     def _path(self, data):
         return data["creator_request"]["path"]
 
-    def _date_updated(self, data):
-        return data["date_updated"]
```

Comment:
I searched the codebase for `\._date_updated` and it was only being used in this module, below.